index.js

```diff
@@ -0,0 +1,3 @@
+const nombreUsuario=prompt('Ingrese su nombre');
+alert(`Hola ${nombreUsuario}!`);
```

Comment:
en este caso me parece mejor no usar interpolacion de string
```suggestion
alert('Hola '+ nombreUsuario);
```